python/cugraph/link_prediction/jaccard_wrapper.pyx

```diff
@@ -114,8 +114,8 @@ def jaccard(input_graph, vertex_pair=None):
             if isinstance(input_graph.edgelist.renumber_map, cudf.DataFrame):
                 unrenumered_df_ = df.merge(input_graph.edgelist.renumber_map, left_on='source', right_on='id', how='left').drop(['id', 'source'])
```

Comment:
Should we change variable to unrenumbered_df_?  Doesn't really matter, but the spelling error distracts me :-(

Reply:
oops, typo, fixing it

Reply:
fixed, thanks